### PR TITLE
MNT Remove deprecated metric arg for metricframe

### PR DESCRIPTION
**Pull request by romanlutz**
## Description
Removing the `metric` arg which was replaced by `metrics` in v0.7.0.

## Tests

- [x] no new tests required
- [ ] new tests added
- [x] existing tests adjusted

## Documentation

- [ ] no documentation changes needed
- [ ] user guide added or updated
- [x] API docs added or updated
- [ ] example notebook added or updated

### CODE REVIEW

**Review by riedgar-ms — APPROVED** [review]
Unless @MiroDudik objects in a timely manner.....

**Comment by romanlutz** [comment]
Thanks for approving @riedgar-ms ! If another maintainer sees this please feel free to merge (if you agree)😄

**Pull request merged**
